tests/test_pytest/test_dtree.py

```diff
@@ -0,0 +1,53 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Takes in a dictionary of xarrays and converts to xarray datatree."""
+
+
+import pytest
+import xarray
+import numpy as np
+from geoips.commandline.log_setup import setup_logging
+from geoips.xarray_utils import xr_to_dtree
+
+LOG = setup_logging()
+
+
+@pytest.fixture
+def load_testfiles():
+    """Generate test xarrays for testing."""
+    # only want a small sample for testing
+    xarray_dict = {
+        "test_{}".format(k): xarray.Dataset(
+            data_vars=dict(temp=(["x", "y"], np.empty((2, 2)))),
+            coords=dict(lon=(["x", "y"], np.empty((2, 2)))),
+        )
+        for k in range(0, 4)
+    }
+    return xarray_dict

```

Comment:
@jsolbrig Is that the sort of thing you are looking for with pytests?

Reply:
Yes I think this looks good. The fixture may not be needed at the moment since it is only getting used once, but I don't see that as a problem. It makes it easier to add other tests later if we want.